app/src/test/java/ar/com/wolox/android/training/ui/login/LogInPresenterTest.java

```diff
@@ -1,3 +1,156 @@
-package ar.com.wolox.android.training;
+package ar.com.wolox.android.training.ui.login;
+import static org.assertj.core.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.eq;
 
-public class LogInPresenterTest {}
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.invocation.InvocationOnMock;
+import org.mockito.stubbing.Answer;
+
+import java.util.LinkedList;
+import java.util.List;
+
+import ar.com.wolox.android.training.ui.network.entities.UserResponse;
+import ar.com.wolox.android.training.ui.network.services.LogInService;
+import ar.com.wolox.wolmo.networking.retrofit.RetrofitServices;
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+
+
+public class LogInPresenterTest {
+
+    RetrofitServices retrofitServicesMock;
```

Comment:
Remember to use Hungarian Notation here. All the properties should start with `m`. Like: `mRetrofitServicesMock`.
I HATE this notation, but it's the one used :disappointed: